src/main/java/org/jenkinsci/plugins/mesos/MesosSlaveInfo.java

```diff
@@ -180,13 +184,27 @@ public String getImage() {
     private final String type;
     private final String dockerImage;
     private final List<Volume> volumes;
+    private final String networking;
+    private final List<PortMapping> portMappings;
 
     @DataBoundConstructor
-    public ContainerInfo(String type, String dockerImage, List<Volume> volumes)
+    public ContainerInfo(String type, String dockerImage, List<Volume> volumes, String networking, List<PortMapping> portMappings)
       throws FormException {
       this.type = type;
       this.dockerImage = dockerImage;
       this.volumes = volumes;
+
+      if (networking == null) {
+          this.networking = Network.BRIDGE.toString();
+      } else {
+          this.networking = networking;
+      }
+
+      if (Network.HOST.equals(Network.valueOf(networking))) {
```

Comment:
@barkerd427 AFAIK now is that HOST mode in Docker exposes the ports of all services in a container on the host network interface. This means that it will not be possible to map ports differently (the ports would have to be configured in the service configuration, for example "Listen 80" in httpd.conf)